README.md

```diff
@@ -47,6 +47,19 @@ SYNTAX:
 INPUT:
         ref_values          List of n reference values
         pred_values         List of n prediciton values
+        title_string        Name of Graph
+        xlabel              Title x-axis (optional)
+        ylabel              Title y-axis (optional
```

Comment:
Good documentation, and looks nice on here, but on the webpage it gets all squished together, can you figure out how to make line breaks?

Also, I might suggest adding the data type for each of the different variables (ie: `xlabel Title x-axis (optional, str)` )